Jonathan/jonathan_library_challenge.py

```diff
@@ -0,0 +1,200 @@
+print('Question 1:')
+# You are working on a library management system, here are the list books at the library
+books = ['MY OWN WORDS', 'WHITE FRAGILITY', 'THE BODY KEEPS THE SCORE', 'SO YOU WANT TO TALK ABOUT RACE', 'STAMPED FROM THE BEGINNING', 'JUST MERCY', 'BORN A CRIME',
+         'THE WARMTH OF OTHER SUNS', 'THE COLOR OF LAW', 'THE NEW JIM CROW', 'THE TRUTHS WE HOLD', 'SAPIENS', 'BRAIDING SWEETGRASS', "MY GRANDMOTHER'S HANDS", 'ON TYRANNY']
+print()
+# 1.0
+# What data type is the object 'books'? How do you know?
+print()
+# It's a string list because of the square brackets and astring list because of the comma's. this can be checked with the type function
+print(type(books))
+print()
+# 1.1
+def available_books():
+    print(books)
+available_books()
+print()
+# Create a function 'available_books' to print the books list
+# Parameters: Not needed for this function
+# Return: Not needed for this function
+
+# 1.2
+# Run the 'available_books' function
+
+# 1.3
+# Create a function 'check_out' that removes a book from the books list
+# Parameters: book (string)
+# Return: Not needed for this functiod
+def check_out(book): #book is paramiter
+    books.remove(book) #removing book from books list
+check_out('SAPIENS')
+print()    
+# 1.4
+# Check out 'SAPIENS' using the check_out function
+
+# Bonus: Run available_books function again to see if the book was checked out
+print(books)
+print('SAPIENS' in books) 
+print()
+# 1.5
+# Create a function 'check_in' that adds a book to the books list
+# Parameters: book (string)
+# Return: Not needed for this function
+def check_in(book):
+    books.append(book) # tell"s the computer which list we are working on
+check_in('SAPIENS')
+print()
+# 1.6
+# Check in 'SAPIENS' using the check_in function
+
+# Bonus: Run available_books function to see if the book was checked in
+print('SAPIENS' in books)
+available_books()
+print()
+# 1.7
+# Create a function 'search_by_name' that prints 'Available' if exists in books list, 'Not Available' if it doesn't.
+def search_by_name(book):
+    if 'SAPIENS':
+     print('Available')
+    else: 
+     print('Not Available')
```

Comment:
Unfortunately, this is incorrect. We are creating a generic function here, so what should be put after the if statement? If you still have no clue, please reach out to me.